cpu/atmega_common/periph/i2c.c

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright (C) 2017 Hamburg University of Applied Sciences, Dimitri Nahm
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup  driver_periph
```

Comment:
should be 

```
 * @ingroup cpu_atmega_common
 * @ingroup drivers_periph_i2c
```

see others, e.g. [spi](https://github.com/RIOT-OS/RIOT/blob/master/cpu/atmega_common/periph/spi.c#L12)